tox.ini

```diff
@@ -20,10 +20,26 @@ commands = nosetests --ignore-files=test_appengine\.py {posargs}
 [testenv:cover]
 basepython = python2.7
 commands =
-    nosetests --with-xunit --with-xcoverage --cover-package=oauth2client --nocapture --cover-erase --cover-tests --cover-branches --ignore-files=test_appengine\.py --cover-min-percentage=60
+    nosetests \
```

Comment:
Since `coveralls` runs

```
nosetests ...
coveralls
```

if the first command fails (the `nosetests` one) then the coverage report will never be sent to `coveralls` since the second command will never run.